### ISSUE TITLE
Remote Relative Includes should search relative to repo root (in addition to purely relative)

### ISSUE THREAD

**Issue by deldrid1** [electricimp/Builder#67]
For a simple remote relative include like
```
@include once "lib/libName/libFile.class.nut"
```
 contained in 
```
@include once "github:SomeUser\someRepo\src\sourceFile.nut"
```
the latest version of Builder does not work...

<hr>

If `github:SomeUser\someRepo\src\sourceFile.nut` contains the following, everything works as expected:
```
@include once "./../lib/libName/libFile.class.nut"
```

This modified behavior is needed because most of my monolithic repos are setup to `@include` files relative to the repo root, rather than relative to the exact file that they are included from.  I could update ALL of my files, but its actually quite a large amount of work and results in much less readable `@include`'s IMHO.  It also makes VSCode's right click __"Copy Relative Path"__ command much less useful...

If instead, a few new unit tests were added and GithubReader.js updated (some `context` information will need to be added in its `read` function as well) to try to search relative to the repo root in addition to the `v2.8.0` functionality, this would be much more flexible.  Builder would only fail if the file wasn't in either location.

**Comment by betzrhodes** [comment]
Looking into this.

**Comment by deldrid1** [comment]
@betzrhodes - any updates here?  Is this something that will be supported?

**Comment by betzrhodes** [comment]
Improving remote relative include functionality has been added to our backlog.

**Comment by alexey-nbl** [comment]
This is addressed by the 4.0.0 release (PR https://github.com/electricimp/Builder/pull/91). Can be closed.
